modules/vaos/app/controllers/vaos/v2/appointments_controller.rb

```diff
@@ -268,10 +264,16 @@ def add_timezone_offset(date, tz)
         utc_date.change(offset: timezone_offset).to_datetime
       end
 
+      FACILITY_ERROR_MSG = 'Error fetching facility details'
```

Comment:
I didn't add this code if you were referencing me 🙃

Reply:
We wanted to provide more specific messaging for when facility info couldn't be retrieved, also meaning the timezone wouldn't be converted. We saw it as providing slightly more value than just checking for nil in this situation, in case the location field is nil for other reasons. The website has plans to surface this message accordingly, but mobile can certainly use it differently if y'all would prefer. Either way, I'll shuffle that constant up in the file 🙂